packages/firebase_analytics/firebase_analytics/example/test_driver/firebase_analytics_e2e.dart

```diff
@@ -30,5 +30,23 @@ void main() {
     expect(
         FirebaseAnalytics().setCurrentScreen(screenName: 'testing'), completes);
     expect(FirebaseAnalytics().logEvent(name: 'testing'), completes);
```

Comment:
Could you add a test for maps inside maps?

Reply:
great stuff so far, @lColinDl 🎉 . I agree with @rrousselGit, we should probably test a nested map as well 👍

Reply:
Thanks!

Could you also add a test for the `throw IllegalArgumentException`?

Reply:
Done